roles/ovirt-repositories/README.md

```diff
@@ -0,0 +1,65 @@
+
+oVirt Repositories
+==================
+
+The `ovirt-repositories` role is used to set the repositories required for oVirt engine installation.
+By default it copies content of /etc/yum.repos.d/ to /tmp/repo-backup-{{timestamp}}.
+Currently it works only when you provide the release RPM.
+
+
+Requirements
+------------
+
+ * oVirt Python SDK version 4
+ * Ansible version 2.4
+
+Role Variables
+--------------
+
+| Name                  | Default value         |  Description                              |
+|-----------------------|-----------------------|-------------------------------------------|
+| ovirt_rpm_repo        | UNDEF                 | URL of RPM package with repository files. |
+| use_subscription_manager| False | If true it will use repos from subscription manager. |
```

Comment:
let's use a prefix for those variables please. usually the prefix is the role name, so we should be consistent across all of your roles.